### PR TITLE
Fix: styling bugs and incorrect spelling

### PR DESCRIPTION
**Pull request by mykalmax**
- [x] [issue 685](https://github.com/TobikoData/sqlmesh/issues/685)

### CODE REVIEW

File: web/client/src/library/components/editor/EditorFooter.tsx [hunk 1]
```diff
@@ -73,7 +73,7 @@ export default function EditorFooter({ tab }: { tab: EditorTab }): JSX.Element {
           </EditorIndicator.Selector>
         </EditorIndicator>
       )}
-      {tab.file.isSQLMeshModel && tab.dialect != null && (
+      {tab.file.isSQLMeshModel && tab.dialect != null && tab.dialect !== '' && (
```
Comment (vchan) [inline web/client/src/library/components/editor/EditorFooter.tsx:76]: Can you do a truthy check here?
```suggestion
      {tab.file.isSQLMeshModel && tab.dialect && (
```

**Review by tobymao — APPROVED** [review]

File: web/client/src/models/file.ts [hunk 2]
```diff
@@ -56,3 +57,9 @@ export class ModelFile extends ModelArtifact<InitialFile> {
     this.content = newContent
   }
 }
+
+function getFileType(path?: string): FileType | undefined {
+  if (path == null || isStringEmptyOrNil(path)) return
+
+  if (path.startsWith('models')) return 'model'
```
Comment (vchan) [inline web/client/src/models/file.ts:64]:
```suggestion
  if (path.startsWith('models')) return FileType.model
```

File: web/client/src/library/components/editor/extensions/index.ts [hunk 3]
```diff
@@ -116,6 +94,41 @@ export function HoverTooltip(models: Map<string, Model>): Extension {
...
+): DecorationSet {
+  const decorations: any = []
+
+  for (const range of view.visibleRanges) {
+    syntaxTree(view.state).iterate({
+      from: range.from,
+      to: range.to,
+      enter({ from, to }) {
+        // In case model name represented in qoutes
+        // like in python files , we need to remove qoutes
```
Comment (vchan) [inline web/client/src/library/components/editor/extensions/index.ts:109]:
```suggestion
        // like in python files, we need to remove qoutes
```

**Review by vchan — APPROVED** [review]

File: web/client/src/library/components/editor/EditorFooter.tsx (hunk 1 above)
Comment (mykalmax) [reply web/client/src/library/components/editor/EditorFooter.tsx:76]: Our Ts config set that we need to explicitly do `tab.dialect != null` to tell TS that `tab` exits for all code below

**Pull request merged**